src/os/ui/query/queryhandler.js

```diff
@@ -267,22 +274,27 @@ os.ui.query.QueryHandler.filters = function(item) {
 
 /**
  * @param {!Object<string, string|boolean>} item
+ * @param {os.query.BaseAreaManager=} opt_areaManager
  * @return {!ol.Feature}
  */
-os.ui.query.QueryHandler.toExclude = function(item) {
-  return /** @type {!ol.Feature} */ (os.ui.areaManager.get(/** @type {string} */ (item['areaId'])));
+os.ui.query.QueryHandler.toExclude = function(item, opt_areaManager) {
+  var am = opt_areaManager ? opt_areaManager : os.ui.areaManager;
+  return /** @type {!ol.Feature} */ (am.get(/** @type {string} */ (item['areaId'])));
 };
 
 
 /**
  * Creates the filter
- *
+ * @param {os.query.BaseQueryManager=} opt_qm
+ * @param {os.query.BaseAreaManager=} opt_am
+ * @param {os.filter.BaseFilterManager=} opt_fm
+ * @param {boolean=} opt_ignoreShown whether or not to filter on shown areas
```

Comment:
This parameter seems like it could use a better name. When true it includes disabled areas, false/undefined it filters out disabled areas. Maybe `opt_includeDisabled`? The default behavior of the function is to only include shown areas.